MenuAPI/MenuController.cs

```diff
@@ -1088,7 +1088,7 @@ internal static async Task DrawInstructionalButtons()
             if (
                 Game.IsPaused ||
                 Game.Player.IsDead ||
-                IsScreenFadedIn() ||
```

Comment:
Almost! It should be !IsScreenFadedIn() to also include when the screen is fading in/out, instead of only when it has already faded out.

Reply:
Ah alright, good point, glad I could help :)